sentry_sdk/hub.py

```diff
@@ -267,16 +268,19 @@ def capture_event(self, event, hint=None):
             return rv
         return None
 
-    def capture_message(self, message, level=None):
-        # type: (str, Optional[Any]) -> Optional[str]
+    def capture_message(self, message, level=None, stacktrace=False):
+        # type: (str, Optional[Any], bool) -> Optional[str]
         """Captures a message.  The message is just a string.  If no level
         is provided the default level is `info`.
         """
         if self.client is None:
             return None
         if level is None:
             level = "info"
-        return self.capture_event({"message": message, "level": level})
+        event = {"message": message, "level": level}
+        if stacktrace is True:
+            event["stacktrace"] = current_stacktrace(self.client.options["with_locals"])
```

Comment:
Possible improvement: Also take a `with_locals` to override the client's options. If `None`, use the client's options?